### PR TITLE
[BACKLOG-28664] Added Map-R 6 CE and EE drivers

### PR DESCRIPTION
**Pull request by angel-ramoscardona**

### CODE REVIEW

**Comment by buildguy** [comment]
### :gear: Build finished in 11m 15s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies -pl \
common-fragment-V1,shims,shims/mapr60,shims/mapr60/driver,shims/mapr60/kar,shims/mapr60/pmr
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **com.pentaho.big.data.bundles.impl.shim.hbase.HBaseServiceImplTest.testGetHBaseConnectionMinimal**
```
    ${result.errorDetails}
```
- **com.pentaho.big.data.bundles.impl.shim.hbase.HBaseServiceImplTest.testGetHBaseConnectionFull**
```
    ${result.errorDetails}
```
  ---

|Total|Failed|Skipped|
|---|---|---|
|757|2|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1016/1/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 10m 19s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies -pl \
common-fragment-V1,shims,shims/mapr60,shims/mapr60/driver,shims/mapr60/kar,shims/mapr60/pmr
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **com.pentaho.big.data.bundles.impl.shim.hbase.HBaseServiceImplTest.testGetHBaseConnectionMinimal**
```
    ${result.errorDetails}
```
- **com.pentaho.big.data.bundles.impl.shim.hbase.HBaseServiceImplTest.testGetHBaseConnectionFull**
```
    ${result.errorDetails}
```
  ---

|Total|Failed|Skipped|
|---|---|---|
|757|2|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1016/2/testReport)|
|----|


:information_source:  _This is an automatic message_

File: common-fragment-V1/pom.xml [hunk 1]
```diff
@@ -34,6 +34,11 @@
                 <zookeeper.version>3.4.5-cdh5.14.0</zookeeper.version>
             </properties>
             <dependencies>
+                <dependency>
+                    <groupId>org.apache.karaf.jaas</groupId>
+                    <artifactId>org.apache.karaf.jaas.config</artifactId>
+                    <version>3.0.3</version>
```
Comment (jgminder) [inline common-fragment-V1/pom.xml]: Isn't the karaf version on the parent pom?

File: common-fragment-V1/pom.xml [hunk 2]
```diff
@@ -415,6 +420,11 @@
           <dependency.pig.classifier>h2</dependency.pig.classifier>
         </properties>
         <dependencies>
+            <dependency>
+                <groupId>org.apache.karaf.jaas</groupId>
+                <artifactId>org.apache.karaf.jaas.config</artifactId>
+                <version>3.0.3</version>
```
Comment (jgminder) [inline common-fragment-V1/pom.xml]: Isn't the karaf version on the parent pom?

File: shims/mapr60/driver/pom.xml [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+    <sqoop.version>1.4.6-mapr-1710</sqoop.version>
+    <zookeeper.version>3.4.5-mapr-1710</zookeeper.version>
+    <vendor-driver.version>6.0.0.0.mapr</vendor-driver.version>
+    <pentaho-code.version>11.0.0.0</pentaho-code.version>
+    <joda-time.version>2.9.9</joda-time.version>
+    <jython.version>2.5.3</jython.version>
+    <org.osgi.core.version>4.3.1</org.osgi.core.version>
+  </properties>
+  <dependencies>
+      <!--
```
Comment (jgminder) [inline shims/mapr60/driver/pom.xml]: Remove the dead dependencies.

File: shims/mapr60/driver/pom.xml [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+          <groupId>org.osgi</groupId>
+          <artifactId>org.osgi.core</artifactId>
+          <version>${org.osgi.core.version}</version>
+          <scope>provided</scope>
+      </dependency>
+      -->
+      <dependency>
+          <groupId>com.mapr.fs</groupId>
+          <artifactId>mapr-hbase</artifactId>
+          <version>6.0.0-mapr</version>
```
Comment (jgminder) [inline shims/mapr60/driver/pom.xml]: Consolidate the versions into properties.

File: shims/mapr60/driver/pom.xml [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+          <groupId>org.apache.hive</groupId>
+          <artifactId>hive-jdbc</artifactId>
+          <version>${org.apache.hive.version}</version>
+      </dependency>
+      <dependency>
+          <groupId>org.apache.hive</groupId>
+          <artifactId>hive-service</artifactId>
+          <version>${org.apache.hive.version}</version>
+      </dependency>
+      <!--
```
Comment (jgminder) [inline shims/mapr60/driver/pom.xml]: Remove the commented dependency.

File: shims/mapr60/driver/pom.xml [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+          <groupId>org.apache.parquet</groupId>
+          <artifactId>parquet-hadoop-bundle</artifactId>
+          <version>${parquet.version}</version>
+      </dependency>
+      <dependency>
+          <groupId>org.apache.pig</groupId>
+          <artifactId>pig</artifactId>
+          <version>${pig.version}</version>
+      </dependency>
+      <!--
```
Comment (jgminder) [inline shims/mapr60/driver/pom.xml]: Remove commented dependencies.

File: shims/mapr60/driver/pom.xml [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+          <groupId>org.python</groupId>
+          <artifactId>jython</artifactId>
+          <version>${jython.version}</version>
+      </dependency>
+  </dependencies>
+
+      <build>
+        <plugins>
+            <!--
+            <plugin>
```
Comment (jgminder) [inline shims/mapr60/driver/pom.xml]: Remove the unnecessary build plugin.

File: shims/mapr60/driver/pom.xml [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+                            <exclude>**/*tests.jar</exclude>
+                        </excludes>
+                        <Import-Package>
+                        </Import-Package>
+                        <_exportcontents>
+                        </_exportcontents>
+                        <Embed-Dependency>*</Embed-Dependency>
+                        <Bundle-Activator>org.pentaho.hadoop.shim.ShimActivator</Bundle-Activator>
+                        <!--
+                        <Bundle-NativeCode>
```
Comment (jgminder) [inline shims/mapr60/driver/pom.xml]: If this isn't necessary remove it.

File: shims/mapr60/driver/pom.xml [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+        <groupId>org.pentaho</groupId>
+        <artifactId>pentaho-hadoop-shims-mapr60-reactor</artifactId>
+        <version>9.0.0.0-SNAPSHOT</version>
+    </parent>
+    <groupId>org.pentaho</groupId>
+    <version>9.0.0.0-SNAPSHOT</version>
+    <artifactId>pentaho-hadoop-shims-mapr60-driver</artifactId>
+  <packaging>bundle</packaging>
+  <properties>
+    <commons-lang.version>2.6</commons-lang.version>
```
Comment (jgminder) [inline shims/mapr60/driver/pom.xml]: Remove any unnecessary properties.

**Comment by buildguy** [comment]
### :gear: Build finished in 29s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies -pl \
common-fragment-V1,shims,shims/mapr60,shims/mapr60/driver,shims/mapr60/kar,shims/mapr60/pmr
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 32s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies -pl \
common-fragment-V1,shims,shims/mapr60,shims/mapr60/driver,shims/mapr60/kar,shims/mapr60/pmr
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by angel-ramoscardona** [comment]
Need to fix building issues

**Pull request closed without merging**